packages/client/public/deploy-config.js

```diff
@@ -20,15 +20,13 @@ window.APP_CONFIG.overrideFeatureFlag = (flagName, overrideValue) => {
 // For local development only. Register feature flags for Staging and Production by
 //  configuring the `website_feature_flags` input variable in Terraform (see `terraform/*.tfvars`).
 window.APP_CONFIG.featureFlags = {
+  myProfileEnabled: true,
   newTerminologyEnabled: true,
-  newGrantsDetailPageEnabled: true,
+  newGrantsDetailPageEnabled: false,
+  categoryOfFundingActivitySearchFieldEnabled: true,
+  shareTerminologyEnabled: true,
```

Comment:
There are a few additional feature flags here that I recognize, but am pretty sure they correspond to projects from a while ago. It's worth mentioning that once a feature is rolled out to Production and deemed stable, we remove any flags associated with that feature, which helps keep things tidy.

I see that this PR is based on a branch from a fork – is it possible that the `featureflag` branch is based on an older version of this repository? I don't see any references to these feature flags in code, so they should not be (re)introduced in this PR:
- `myProfileEnabled`
- `categoryOfFundingActivitySearchFieldEnabled`

Similarly, the `newGrantsDetailPageEnabled` should remain set to `true` (this PR changes it to `false`).